src/main/java/com/example/Library/services/UserService.java

```diff
@@ -0,0 +1,23 @@
+package com.example.Library.services;
+
+import com.example.Library.model.Book;
+import com.example.Library.model.User;
+import com.example.Library.repositories.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+@Service
+public class UserService {
```

Comment:
if AuthenticationService introduced, there will be no need in UserService as it will not be used from any controller